README.md

```diff
@@ -130,6 +130,19 @@ We are currently working on three different proof backends:
 	- SMT theorems
 
 
+The types of proofs that can be performed by the various backend can be understood in the following table:
+
+| Proof type \ Backend          | HEVM    | KEVM | SMT     | Coq     |     |
+| ----------------------------- | ------- | ---- | ------- | ------- | --- |
+| Bytecode (with unknown calls) | Planned | ?    |         |         |     |
+| Bytecode                      | Planned | WIP  |         |         |     |
+| Poststate => postcondition    |         |      | Planned |         |     |
+| Source level                  |         |      | Planned |         |     |
+| Contract level invariant      |         |      |         | Planned |     |
```

Comment:
```suggestion
| Contract level invariant      |         |      | Planned | Planned |     |
```
?

Reply:
What's the rightmost column?

Reply:
hmmm, a mistake I think

Reply:
I think the right most column is supposed to be coq, and the penultimate column from the right is not supposed to exist

Reply:
Wait, it still has this rightmost column with empty header

Reply:
yes, sorry, this shouldn't be marked as resolved